### DISCUSSION TITLE
FR: Craft 5: Add an alert while editing the source field through the edit drawer

### DISCUSSION THREAD

**Discussion by Stalex89** [craftcms/cms#16261] [Ideas] [answered]
Sometimes instead of editing the "instance" of the field in the entry by pressing the "Settings" options, we are accidentally editing the "source" field via the "Edit field option" which causes the changes throughout the entire site. Could it be possible to add some alert saying "You about to modify the field blueprint/source and not the field instance. That may affect the entire website. Proceed?". Just a QoL improvement but may save tons of nerves and rollback time. Thank you in advance!

![Screenshot 2024-12-03 at 10 54 59](https://github.com/user-attachments/assets/d7cb76d3-7555-4264-bb17-d45e62ea43cc)

**Comment by brandonkelly** [comment, marked as answer]
Just reworded those two action labels for Craft 5.6, to hopefully make it a bit more clear: (c66045bf71c6432bcf48c88d049b4b1e2735fbd5)

- “Instance settings”
- “Edit global field settings”

<img src="https://github.com/user-attachments/assets/1e8dad59-eece-43e8-b85b-64997384a57b" width="279" height="126" alt="A field instance’s action menu, showing the new action labels">

**Reply by Stalex89** [reply]
Super, thank you for the improvement! 🙂

**Reply by brandonkelly** [reply]
Craft 5.6.0 is out now! I ended up removing the global field settings item entirely, since it’s accessible from the instance settings at the top of the slideout. (#16451)

**Reply by Anubarak** [reply]
@brandonkelly Could you please consider to include an option to show it again?

Of course you can edit it by opening a second overlay and that's great, but one advantage in Craft 5 was to have this overlay directly available in the field layout designer.
As it is now I always have to double click 2 times which is no problem once or twice but makes it worse when you want to use that every day.

Same goes for sections -> entry types. It was so great to see all fields of the entry types within one double click and edit a field directly.

Now I have to open 4 sidebars and click way more often.
I frequently end up with a stack like this now
![image](https://github.com/user-attachments/assets/d299cefa-57d8-442f-8806-894543eae645)
